client/src/components/Footer/FooterRight/About/index.tsx

```diff
@@ -0,0 +1,11 @@
+import { Typography } from '@mui/material';
+import { TiteledPargraph } from '../../components.styled';
+
+export const About = () => (
+  <TiteledPargraph>
+    <Typography component="span">About The Company</Typography>
+    <strong>Saed Developer</strong> A web developer where you can find more
```

Comment:
what
this is not your portfolio page

Reply:
Yop I know, i put it instead of Lorem Ipsum for now hhhh